opcUaDevSupApp/Makefile

```diff
@@ -27,8 +27,13 @@ USR_INCLUDES += -I$(UASDK)/include/uaclient
 USR_INCLUDES += -I$(UASDK)/include/uapki
 USR_INCLUDES += -I$(UASDK)/include/uastack
 USR_INCLUDES += -I$(UASDK)/include/xmlparser
+USR_INCLUDES += -I$(BOOST)
```

Comment:
That should be only added if BOOST is defined - else it will create an invalid commandline option.